src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -860,6 +866,218 @@ else if (configJson.getString("metadata").equals("xmp")) {
+            //Creation JSON
+            File convertedFile = new File(properties.getProperty("file.conversionDir") +
+                outputJson.getString("name") + "/" + outputJson.getString("name") +
+                outputJson.getString("ext"));
+            Metadata metaDataFile = new Metadata();
+            metaDataFile.writeJsonFile(convertedFile);
+        }
+        else if (configJson.getString("metadata").equals("xmp")) {
+            //Creation XMP
+            File convertedFile = new File(properties.getProperty("file.conversionDir") +
+                outputJson.getString("name") + "/" + outputJson.getString("name") +
+                outputJson.getString("ext"));
+            Metadata metaDataFile = new Metadata();
+            metaDataFile.writeXmpFile(convertedFile);
+        }
+        if (configJson.getBoolean("thumbnail")) {
+            //Creation thumbnail
+            CriteriaPdfToImage pdfToImgCriteria = new CriteriaPdfToImage();
+            pdfToImgCriteria.setSrcPath(properties.getProperty("file.uploadDir") + fileName);
+            pdfToImgCriteria.setDestPath(properties.getProperty("file.conversionDir") + outputJson.getString("name") +
+                "/");
+            pdfToImgCriteria.setName(outputJson.getString("name"));
+            pdfToImgCriteria.setExt(".jpg");
+            pdfToImgCriteria.setDpi(150);
+            pdfToImgCriteria.setFormatColor("RGB");
+            PdfThumbnail pdfToThumb = new PdfThumbnail();
+            pdfToThumb.convert(pdfToImgCriteria);
+        }
+
+        // Zip all the files in the conversion directory
+        FolderZipped.zipFolder(properties.getProperty("file.conversionDir") + outputJson.getString("name"));
+
+        // If a target location has been specified, copy the zip file there.
+        if (outputJson.has("destPath")) {
+            File resultZip = new File(properties.getProperty("file.conversionDir") + outputJson.getString("name") + ".zip");
+            File targetZip = new File(outputJson.getString("destPath") + outputJson.getString("name") + ".zip");
+            try {
+                copyFile(resultZip, targetZip);
+            }
+            catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        return new PPTtoPdfResponse(fileName, fileDownloadUri, outputChecksumString);
+    }
+
+    /**
+     * Converts from a PDF document to to a MS Word document based on a user-provided criteria: input, config and output.
+     *
+     * @param asset The input pdf document itself
+     * @param input A JSON formatted string (wo \n characters) that holds all the parameters tied to the input document
+     * @param config A JSON formatted string (wo \n characters) that holds all the parameters tied to the configuration
+     *               of the particular process of conversion.
+     * @param output A JSON formatted string (wo \n characters) that holds all the parameters tied to the output document
```

Comment:
At the end of the comment, you missed the point "."

Reply:
Fixed